src/auctions.test.ts

```diff
@@ -62,58 +60,50 @@ describe('calculateAuctionPriceForBlock', () => {
     it.each([
       // we keep the scalingComponent consistent to make it easier to reason about the test cases, and to represent the decay in the auction curve for block heights and varying decay rates
       [[0, 0, 0.001, 90], 100],
-      [[0, 1, 0.001, 90], 91.389003],
-      [[0, 2, 0.001, 90], 83.511968],
-      [[0, 3, 0.001, 90], 76.306977],
+      [[0, 1, 0.001, 90], 99.962007],
+      [[0, 2, 0.001, 90], 99.924029],
+      [[0, 3, 0.001, 90], 99.886065],
       [[0, 0, 0.002, 90], 100],
-      [[0, 1, 0.002, 90], 83.511968],
-      [[0, 2, 0.002, 90], 69.717284],
-      [[0, 3, 0.002, 90], 58.180118],
+      [[0, 1, 0.002, 90], 99.962007],
+      [[0, 2, 0.002, 90], 99.924029],
+      [[0, 3, 0.002, 90], 99.886065],
       // block heights before the start height should just return the start price
       [[10, 9, 0.001, 90], 100],
       [[10, 0, 0.001, 90], 100],
     ])(
       'given [current block height, moving average purchase count] of %j, should return %d',
-      (
-        [startHeight, currentHeight, exponentialDecayRate, scalingExponent],
-        expectedPrice,
-      ) => {
+      ([startHeight, currentHeight], expectedPrice) => {
```

Comment:
we no longer stub in these decay rates and scaling exponents. so our expectation changes

Reply:
nit: should also remove stubs from the array above